### PR TITLE
docs: add local development environment section to README

### PR DESCRIPTION
**Pull request by radnov**
The intent is to have a minimal step-by-step guide for starting a local development environment (albeit not fully functional).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+
+1. Start by copying the `.env.example` file into a new `.env` file:
+```
+cp .env.example .env
+```
+2. Create a private key:
+```
+make keys
+```
+3. Copy the private key contents into the `PRIVATE_KEY` environment variable within the `.env` file:
```
Comment (tonsV2) [inline README.md:17]: Maybe describe what needs to be done? Replace actual newlines with \n

Comment (tonsV2) [reply README.md:17]: Maybe we should even let the make keys command output: PRIVATE_KEY="..." ?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+4. Initialize the environment and install dev dependencies:
+```
+make init
+```
+5. Start a development environment:
+```
+make dev
+```
 
 # Add a group
```
Comment (tonsV2) [inline README.md:31]: This now seems a bit unrelated. Should we just get rid of it?

**Review by tonsV2 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (radnov) [reply README.md:17]: >Maybe we should even let the make keys command output: PRIVATE_KEY="..." ?
I didn't do that because dumping the private key contents on the screen didn't seem like a good practice and copy/pasting from the terminal could also lead to unexpected newlines.
